22-Drawing App/script.js

```diff
@@ -0,0 +1,85 @@
+const canvas = document.getElementById('canvas')
+const increaseBtn = document.getElementById('increase')
+const decreaseBtn = document.getElementById('decrease')
+const sizeEl = document.getElementById('size')
+const colorEl = document.getElementById('color')
+const clearEl = document.getElementById('clear')
+
+const ctx = canvas.getContext('2d')
+
+let size = 10
+isPressed = false
+let color = 'black'
+let x
+let y
+
+canvas.addEventListener('mousedown', (e) => {
+  isPressed = true
+
+  x = e.offsetX
+  y = e.offsetY
+})
+
+document.addEventListener('mouseup', (e) => {
+  isPressed = false
+
+  x = undefined
+  y = undefined
+})
+
+canvas.addEventListener('mousemove', (e) => {
+  if (isPressed) {
+    const x2 = e.offsetX
+    const y2 = e.offsetY
+
+    drawCircle(x2, y2)
+    drawLine(x, y, x2, y2)
+
+    x = x2
+    y = y2
+  }
+})
+
+function drawCircle(x, y) {
+  ctx.beginPath();
+  ctx.arc(x, y, size, 0, Math.PI * 2);
+  ctx.fillStyle = color
+  ctx.fill()
+}
+
+function drawLine(x1, y1, x2, y2) {
+  ctx.beginPath()
+  ctx.moveTo(x1, y1)
+  ctx.lineTo(x2, y2)
+  ctx.strokeStyle = color
+  ctx.lineWidth = size * 2
+  ctx.stroke()
+}
+
+function updateSizeOnScreen() {
+  sizeEl.innerText = size
+}
+
+increaseBtn.addEventListener('click', () => {
+  size += 5
```

Comment:
but what is 5, why 5? why not 6, 7, etc? 

These are called Magic numbers, and they are generally bad practice, Here is an article for more information: https://blog.webdevsimplified.com/2020-02/magic-numbers/